src/components/Gradebook/index.jsx

```diff
@@ -503,13 +543,57 @@ export default class Gradebook extends React.Component {
               </div>
               {this.props.showBulkManagement && (
                 <div>
+                  <h4>Step 1: Filter the Grade Report</h4>
```

Comment:
We could do that, but since we're going to tear out this tab and reduce back down to one tab having filters later it may not be worth it.

Additionally it's probably better to separate off another component (since that's the major means of abstracting code in the world of React), rather than using a sub-function to split off JSX used in common.